pkg/storage/mvcc.go

```diff
@@ -5866,9 +5866,9 @@ func MVCCExportToSST(
 	}
 
 	if summary.DataSize == 0 {
-		// If no records were added to the sstable, skip completing it and return a
-		// nil slice – the export code will discard it anyway (based on 0 DataSize).
-		return roachpb.BulkOpSummary{}, MVCCKey{}, nil
+		// If no records were added to the sstable, skip
+		// completing it and return an empty summary.
```

Comment:
Perhaps add something about why we're propagating the resume key (what you've explained in the commit message -- we might still want resumption).